api/proto/teleport/legacy/types/types.proto

```diff
@@ -437,10 +468,13 @@ message AWS {
     (gogoproto.jsontag) = "opensearch,omitempty"
   ];
 
-  // IAMPolicyExists indicates whether the IAM Policy is configured properly for database access.
+  reserved 13;
+  reserved "IAMPolicyExists";
```

Comment:
i thought it was okay to completely remove it, it isn't being consumed anywhere

Reply:
Were these backported to v13 and released in any of the patch releases? If not, we can just remove these, if they were then it's safer to keep them reserved.

Reply:
> Were these backported to v13 and released in any of the patch releases

yes to 13.2.3 and 13.2.5. and i agree (sorry i meant "completely" remove by not deprecating it first)